man/rgbasm.5

```diff
@@ -159,8 +158,10 @@ will be expanded in all of
 .Ql PURGE {name} ,
 and
 .Ql MACRO {name} ,
-but, for example, won't be in
-.Ql DEF(name) .
+even though it won't be in
+.Ql DEF(name) ,
+.Ql PURGE {name} ,
+etc.
 .Pp
 It's possible to change the way symbols are printed by specifying a print format like so:
 .Ql {fmt:symbol} .
```

Comment:
(Looking to comment on one of the lines below, but can't for technical reasons.)

I think we should say “specifier consists of parts, which must be in the following order:” or something like that.

Also, I think we should move the “all parts are optional” statement below to here, to reduce confusion.